chia/util/db_wrapper.py

```diff
@@ -11,12 +11,10 @@ class DBWrapper:
 
     db: aiosqlite.Connection
     lock: asyncio.Lock
-    allow_upgrades: bool
     db_version: int
 
-    def __init__(self, connection: aiosqlite.Connection, allow_upgrades: bool = False, db_version: int = 1):
+    def __init__(self, connection: aiosqlite.Connection, db_version: int = 1):
```

Comment:
If my above comment was correct, this should also not be a change.

Reply:
There's a couple other instances spread throughout the PR

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/9768